.github/workflows/static_analysis.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: |
           pids=()
 
-          TAG=$(cat .terraform-version)
+          TAG=1.5.6
```

Comment:
I think we should update .terraform-version instead.

Reply:
fix present in https://github.com/pagopa/io-infra/commit/44102f1819c531a50be0f27a72a7ebc60824391e